npm/private/npm_translate_lock.bzl

```diff
@@ -499,6 +501,10 @@ def npm_translate_lock(
 
         pnpm_version: pnpm version to use when generating the @pnpm repository. Set to None to not create this repository.
 
+            Deprecated for bzlmod, use `pnpm_extension` instead.
```

Comment:
Should the docs be `pnpm_version` for WORKSPACE and `pnpm_extensions` for MODULE.bazel?

We should fail if you set `pnpm_version` in your MODULE.bazel if that is the case

Reply:
Ideally `pnpm_version` gets dropped from the bzlmod extension. Today it's a noop, so do you think it's ok to actually drop it even though it's sort of a "breaking change"?

Reply:
But the docs for bzlmod + WORKSPACE are both from this same snippet...

Reply:
As long as you say which parameter is for which then the docs being common is fine IMO. Long term WORKSPACE will be gone and this will get cleaned up.

Reply:
I updated this one 👍